test/e2e/route-access/admin.e2e-spec.ts

```diff
@@ -89,6 +89,12 @@ describe('Admin Routes can be accessed by an admin user', () => {
         .delete(`${USER_ROUTE_PREFIX}/${newUser.id}`)
         .expect(200);
     });
+
+    it('/dummy (GET)', async () => {
```

Comment:
Is this needed for Sonar? Can't we exclude it somehow instead of writing a dummy get method?

Reply:
@Saela Yes, this was just for testing whether Sonar picks up integrationtest coverage as well. That's why this is in a separate branch and in a closed PR :D Sorry for adding you as a reviewer, that was not required 😆